### ISSUE TITLE
Support syntax highlighting implementations based on whether they pass all test cases for given keywords

### ISSUE THREAD

**Issue by Julian** [bowtie-json-schema/bowtie#862]
This is low priority / longer term, especially given it likely depends on #27 and maybe #24 as well as some general library for semantically understanding schemas -- but nonetheless, for that day in the future:

When running `bowtie validate` (or any other ad hoc validation) and comparing implementation results, it would be helpful to see whether an implementation is known to be "fully compliant for the given schema" or not -- in other words, given a schema `S`, if the implementation passes the test suite for all keywords contained (recursively) in S, call it "compliant" and highlight it differently in the terminal (and possibly UI). This makes it easier to see when implementations differ on results even though they pass the suite.

#23 is likely also relevant in that maybe it can make use of this information as *input*.
